packages/cyberstorm/src/components/Tooltip.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { PropsWithChildren } from "react";
+import * as RadixTooltip from "@radix-ui/react-tooltip";
+import styles from "./componentStyles/Tooltip.module.css";
+
+export interface TooltipProps {
+  label?: string;
+}
+
+// Cyberstorm Tooltip component
+
+export const Tooltip: React.FC<PropsWithChildren<TooltipProps>> = ({
+  label,
+  children,
+}) => {
+  return (
+    <RadixTooltip.Provider delayDuration={150}>
```

Comment:
Is this provider something that has to be included for every tooltip individually, or should it be included once globally?

Reply:
It at least doesn't work without it. I think it can be defined once globally, but the question is; do want to do that? Cause then the app that's using CS would need to define the provider. If I've understood correctly. Let me know if there is a way to do this, that I'm not aware of.

Reply:
> I think it can be defined once globally, but the question is; do want to do that?

That is the question I'm asking. If it can be defined once globally without negative side effects to it's usage, then yes we should do that as it's more efficient. However, it could also be that the API of Radix requires that each tooltip has an unique context.

So refer to the docs and/or test around. If a single global one works, that should be used.

> Cause then the app that's using CS would need to define the provider.

Yeah, this is standard practice; we will have multiple providers at the root level of all apps in every case. What we might want to do is wrap them all into a single component on the CS component library, so you'll need to add a single `CyberstormProvider` at the root of the app which includes everything necessary. We'll need to do this for global CSS variables and such at least so it wouldn't be hard to bundle both.

TL;DR: Check if a global provider can be used, and if there are downsides to it. If it works equally well, then let's use that pattern as it's more efficient.

Reply:
Alright, so I've now added a `Providers.tsx` file which houses the `CyberstormProviders`. The `RadixTooltip.Provider` is defined in there.

Still I do have some performance hit worries with this bundling of all the providers together. To my understanding, all the components within the app will be force updated, if for example the tooltips child is updated. Probably not a problem, that cant be fixed later, but something to keep in mind.

TLDR; provider is now global and working 👍